src/openrct2/interface/Window.h

```diff
@@ -352,4 +350,6 @@ namespace OpenRCT2
     void WindowFollowSprite(WindowBase& w, EntityId spriteIndex);
     void WindowUnfollowSprite(WindowBase& w);
 
+    int16_t GetTitleBarHeight();
+    int16_t GetTitleHeightDiff();
```

Comment:
Let’s use the new code style for new methods (considering you don’t like reviewing code style PR either, and it has to happen *somewhere*).

Reply:
```suggestion
    int16_t getTitleBarHeight();
    int16_t getTitleHeightDiff();
```